### DISCUSSION TITLE
how i do a stage

### DISCUSSION THREAD

**Discussion by jasonroni** [YoshiCrafter29/YoshiCrafterEngine#96] [Q&A] [answered]
The Tittle Says It

**Comment by Furamona** [comment, marked as answer]
var stage:Stage = null;

function create() {
    stage = loadStage("stage"); // Replace stage by the name of your stage file without the extension
}

function beatHit(curBeat) {
    stage.onBeat();
}

**Reply by jasonroni** [reply]
forgot it,the 1.7.0 solved it but i put as an answer cuz yes
